fastlane/Fastfile

```diff
@@ -30,27 +30,29 @@ end
 
 desc "Release a new version"
 lane :release do |options|
-  ensure_git_branch(branch: 'main') # We can only release on default branch
```

Comment:
I would still keep it, just in case. We don't want to find us in a situation where we fuck it up and we release from another branch